iphone/Classes/KrollContext.m

```diff
@@ -1610,36 +1622,54 @@ - (void)dealloc
 
 - (void)invalidateAllTimers
 {
-  for (NSNumber *timerIdentifier in self.timers.allKeys) {
+  for (NSNumber *timerIdentifier in self.timers) {
```

Comment:
Is it save to iterate directly over the NSMapTable and remove entries?

Reply:
ah good catch.